### PR TITLE
Using jovial_svg with pre-compiled binaries, for perfomrance improvment.

### PR DESCRIPTION
**Pull request by iamriajul**
This PR contains performance improvements, and memory and storage footprint improvements.
And also added customizability.

### CODE REVIEW

File: lib/circle_flags.dart [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
-            height: size,
-            child: const CircularProgressIndicator(),
-          );
-        },
+      child: ScalableImageWidget.fromSISource(
+        si: ScalableImageSource.fromSI(
+            DefaultAssetBundle.of(context),
+            'packages/circle_flags/assets/jovial/${countryCode.toLowerCase()}.si'
+        ),
+        onLoading: onLoading ?? _buildPlaceholder,
```
Comment (cedvdb) [inline lib/circle_flags.dart:34]: In my opinion the place holder should be an empty container by default. It's really not that long until it shows up and having something blinking is more annoying

File: lib/circle_flags.dart [hunk 2]
```diff
@@ -1,76 +1,78 @@
...
+        onError: _buildPlaceholder,
```
Comment (cedvdb) [inline lib/circle_flags.dart:35]: There should be an error view at least so the user can correct any misconfiguration

**Review by cedvdb — COMMENTED** [review]
Have you tried and noticed non negligeable performance improvements with the jovial implementation ?

**Comment by iamriajul** [comment]
> 
I'm closing this PR, I should have benchmarked myself first before working jovial_svg. flutter_svg outperforms very easily jovial_svg.
In my test flutter_svg was able to keep an average FPS of 58, but the jovial_svg couldn't it was below 55.

**Pull request closed without merging**